src/server/system_services/bucket_server.js

```diff
@@ -1132,6 +1132,26 @@ function update_bucket_lambda_trigger(req) {
         .return();
 }
 
+
+async function update_all_buckets_default_pool(req) {
+    const pool_name = req.rpc_params.pool_name;
+    const pool = req.system.pools_by_name[pool_name];
+    if (!pool) throw new RpcError('INVALID_POOL_NAME');
+    const internal_pool = pool_server.get_internal_mongo_pool(pool.system);
+    if (String(pool._id) === String(internal_pool._id)) return;
```

Comment:
I would at least print a log warning on that.
Also in general this function is so rare that I would be more verbose and log an info message with details on the pools and the buckets to be modified.

Reply:
added a small print we can edit if more information needed - although most of the info in printed in the make_changes